app/services/album.js

```diff
@@ -0,0 +1,21 @@
+const rp = require('request-promise'),
```

Comment:
It would be better to use a better variable name, like `request` or `request-promise`